docs/setup.rst

```diff
@@ -1,32 +1,29 @@
+=====
 Setup
 =====
 
-Need to set up Conda environment first and foremost
-
-If on workstation:
-module load conda
+Set up Conda environment
+========================
 
-Create new Conda environment
-conda create -n [environmentName]
+various options work quite well:
 
-Append necessary channels
-conda config --append channels noaa-gfdl
-conda config --append channels conda-forge
 
-Run conda install on needed dependencies
-conda install noaa-gfdl::fre-cli should install the CLI package located at https://anaconda.org/NOAA-GFDL/fre-cli created from the meta.yaml file
+If on workstation
+-----------------
 
-All other dependencies used by the tools are installed along with this install (configured inside the meta.yaml), with the exception of local modules
-setup.py file allows fre.py to be ran with fre as the entry point on the command line instead of python fre.py
+``module load fre/canopy``
```

Comment:
I thought this was changing to `fre/2024.01`

Reply:
@ceblanton ?